src/System.Reflection.Metadata/src/System/Reflection/Internal/Utilities/FileStreamReadLightUp.cs

```diff
@@ -142,10 +142,11 @@ internal static unsafe bool TryReadFile(Stream stream, byte* buffer, long start,
             return true;
         }
 
+#pragma warning disable BCL0015 
         private static unsafe class NativeMethods
         {
             // API sets available on modern platforms:
-            [DllImport(@"api-ms-win-core-file-l1-1-0.dll", EntryPoint = "ReadFile", ExactSpelling = true, SetLastError = true)]
+            [DllImport(@"api-ms-win-core-file-l1-2-0.dll", EntryPoint = "ReadFile", ExactSpelling = true, SetLastError = true)]
```

Comment:
What's the difference between the ReadFile exposed from l1-1-0.dll vs l1-2-0.dll?  I'd have thought that they were the same and that the 1 version would be available in more places, so there'd be only upside and no downside to using it.  Is that not the case?